services/frontend/src/i18n/en_CA.json

```diff
@@ -272,7 +272,7 @@
   "rules.required": "Required",
   "rules.url": "Must be a URL address ex—https://www.google.com",
   "rules.valid.phone.number": "Must be a valid phone number",
-  "rules.valid.pri": "Must be a valid PRI",
+  "rules.valid.pri": "Must be a valid PRI ex—23241235",
```

Comment:
I think we should change it to: Must be a valid PRI (e.g. 22222222)
* also i think we should give them a fake number that they can't actually use. in this case 22222222 is the correct number of digits but the validator wont accept it so the person can't use it to fake their PR)